harper-quarto/src/lib.rs

```diff
@@ -0,0 +1,30 @@
+use harper_core::{
+    Lrc, Token,
+    parsers::{Markdown, MarkdownOptions, Mask, Parser},
+};
+
+mod masker;
+pub use masker::QuartoMasker;
+
+/// Parses a Literate Haskell document by masking out the code and considering text as Markdown.
```

Comment:
I'm guessing this comment should say "Quarto document" instead of "Literate Haskell document".